components/canvas.py

```diff
@@ -167,7 +171,7 @@ def mouse_to_3d(self, x, y, z=1.0, local_transform=False):
         glGetIntegerv(GL_VIEWPORT, viewport)
         glGetDoublev(GL_PROJECTION_MATRIX, pmat)
         glGetDoublev(GL_MODELVIEW_MATRIX, mvmat)
-        gluUnProject(x, y, z, mvmat, pmat, viewport, px, py, pz)
+        gluUnProject(x, y, z, mvmat, pmat, viewport)#, px, py, pz)
```

Comment:
gluUnProject will store the results in px, py and pz. Why do they have to be commented out?

Reply:
Sorry, this was just for testing, removing the comment